examples/bri-3/prisma/prisma.service.ts

```diff
@@ -1,15 +1,9 @@
-import { Injectable, OnModuleInit, INestApplication } from '@nestjs/common';
+import { Injectable, OnModuleInit } from '@nestjs/common';
 import { PrismaClient } from '@prisma/client';
 
 @Injectable()
 export class PrismaService extends PrismaClient implements OnModuleInit {
   async onModuleInit() {
     await this.$connect();
   }
-
```

Comment:
Thank you for removing this. We should also create a nestjs module for prisma. I beleive the way its being injected now we have a different instantiation of prisma for each new connection. If we create the nestjs prisma module, that should no longer be the case.

Reply:
I think that by default nestjs provider is a singleton. But i agree we shoudl extract to a separate module, it is cleaner. Could you open a issue for mil5?